LongRunning/composer.json

```diff
@@ -3,6 +3,7 @@
     "description": "Google LongRunning Client for PHP",
     "license": "Apache-2.0",
     "minimum-stability": "stable",
+    "version": "0.2.6",
```

Comment:
composer needs to know the `LongRunning` version in order to install the `google/gax` dependency, as this is used to resolve the circular dependency in `google/gax` which in turn _requires_ `google/longrunning`

Reply:
See https://github.com/googleapis/release-please/pull/2192

Reply:
Resolving the comment as this change is not present there anymore.